MediaBrowser.Api/VideosService.cs

```diff
@@ -138,32 +138,15 @@ public void Post(MergeVersions request)
             var videosWithVersions = items.Where(i => i.MediaSourceCount > 1)
                 .ToList();
 
-            var primaryVersion = videosWithVersions.FirstOrDefault();
-
-            if (primaryVersion == null)
-            {
-                primaryVersion = items.OrderBy(i =>
-                    {
-                        if (i.Video3DFormat.HasValue)
-                        {
-                            return 1;
-                        }
-
-                        if (i.VideoType != Model.Entities.VideoType.VideoFile)
-                        {
-                            return 1;
-                        }
-
-                        return 0;
-                    })
-                    .ThenByDescending(i =>
-                    {
-                        var stream = i.GetDefaultVideoStream();
-
-                        return stream == null || stream.Width == null ? 0 : stream.Width.Value;
-
-                    }).First();
-            }
+            var primaryVersion = videosWithVersions.FirstOrDefault()
+                                 ?? items.OrderBy(i =>
+                                         i.Video3DFormat.HasValue
+                                             ? 1
+                                             : i.VideoType != Model.Entities.VideoType.VideoFile
+                                                 ? 1
+                                                 : 0)
```

Comment:
I think the `if` statements were easier to parse than this nested ternary, even if it's much shorter

Reply:
done